specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1980,6 +1980,9 @@
         "autoScalerProfile": {
           "type": "object",
           "properties": {
+            "balance-similar-node-groups": {
```

Comment:
I believe this is a breaking change requiring a new api-version by ARM standards.  Looping in ARM folks to verify

Reply:
yup, we're consolidating all of our changes against a dev branch then we'll target master with the new api version.